ui/apps/platform/cypress/integration/vulnerabilities/vulnerabilities.helpers.js

```diff
@@ -0,0 +1,10 @@
+import { selectors } from './vulnerabilities.selectors';
+
+export function clearSearchFilters() {
+    cy.get('body').then((body) => {
+        if (body.find(selectors.clearFiltersButton).length > 0) {
```

Comment:
I added this conditional because I found that there were no default filters for me locally. I guess we can assume that they are set in CI, but I'll leave it like this to make it easier

Reply:
I see what you mean. But isn't the adding and removing of filters something that doesn't depend on any sort of data-loading? It may affect data-loading, but I don't think the opposite is true. The filters do depend on reading the URL search though so maybe we can check to see if the search input is rendered.

Reply:
The removal of filters will depend on data-loading, since once the filters are cleared it will refresh the gql request and there could be a couple of second delay for the new data to be available.

Adding filters doesn't depend on data-loading, but it does depend on a `useEffect` that syncs the value found in local storage with the page URL. Due to this, the page will render once without the filters, then the effect will fire, and then there will be a second render that does include the filters.